bindata/linuxptp/ptp-daemon.yaml

```diff
@@ -23,63 +23,106 @@ spec:
     spec:
       hostNetwork: true
       hostPID: true
+      dnsPolicy: ClusterFirstWithHostNet
       nodeSelector:
         kubernetes.io/os: linux
       tolerations:
       - operator: Exists
       serviceAccountName: linuxptp-daemon
       priorityClassName: "system-node-critical"
       containers:
-      - name: kube-rbac-proxy
-        image: {{.KubeRbacProxy}}
-        imagePullPolicy: IfNotPresent
-        args:
-          - --logtostderr
-          - --secure-listen-address=:8443
-          - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
-          - --upstream=http://127.0.0.1:9091/
-          - --tls-private-key-file=/etc/metrics/tls.key
-          - --tls-cert-file=/etc/metrics/tls.crt
-        ports:
-          - containerPort: 8443
-            name: https
-        resources:
-          requests:
-            cpu: 10m
-            memory: 20Mi
-        terminationMessagePolicy: FallbackToLogsOnError
-        volumeMounts:
-          - name: linuxptp-certs
-            mountPath: /etc/metrics
-            readOnly: True
-      - name: linuxptp-daemon-container
-        securityContext:
-          privileged: true
-        image: {{.Image}}
-        command: [ "/bin/bash", "-c", "--" ]
-        args: [ "/usr/local/bin/ptp --alsologtostderr -v 10" ]
-        imagePullPolicy: IfNotPresent
-        resources:
-          requests:
-            cpu: 20m
-            memory: 50Mi
-        env:
-        - name: NODE_NAME
-          valueFrom:
-            fieldRef:
-              fieldPath: spec.nodeName
-        volumeMounts:
-        - name: config-volume
-          mountPath: /etc/linuxptp
-        - name: socket-dir
-          mountPath: /var/run
+       {{ if (eq .EnableEventPublisher true) }}
+        - name: cloud-native-event-sidecar
+          image: {{ .SideCar }}
+          args:
+            - "--metrics-addr=127.0.0.1:9091"
+            - "--store-path=/store"
+            - "--transport-host={{ .EventTransportHost }}"
+            - "--api-port=8080"
+          volumeMounts:
+            - name: pubsubstore
+              mountPath: /store
+            - name: logs-socket-dir
+              mountPath: /tmp
+          ports:
+            - name: metrics-port
+              containerPort: 9091
+          env:
```

Comment:
I think better to have all this in a config map that the event sidecar will use. this way in the future if we need other fields there is no need to change the ptp operator just the sidecar code

Reply:
This is the same as LinuxPTPDaemon. I am just moving metrics from LinuxPTPDaemon to Sidecar when EventPublisher is enabled. trying to keep it consistent with the metric endpoint when we move it to the sidecar

Reply:
what you mean here can you please explain me again?

I think you can just use 

```
 - name: config-volume
          configMap:
            name: ptp-configmap
```

and use it to get the informantion you need for the sidecard

Reply:
So by default, you have the same metrics configured for daemon right, and then we have kube-rbac-proxy configure to point to 127.0.0.1:9091. Putting in configmap and allowing the user to change metrics endpoint, will also need to change kube-rbac-proxy? is that correct? I don't see any value in changing this to config map.